app/api/routes/google.py

```diff
@@ -0,0 +1,73 @@
+# import os
+from typing import Any
+
+from authlib.integrations.starlette_client import OAuth  # type: ignore
+from fastapi import APIRouter, Request, status
+from fastapi.responses import HTMLResponse
+
+from app.api.routes.users import create_user_inner
+from app.core.config import settings
+from app.models.user import UserCreate
+from app.utilities.dependencies import SessionDep
+from app.utilities.exceptions import MissingFieldException
+from app.utilities.messages import GOOGLE_RESPONSES
+
+router = APIRouter()
+
+oauth = OAuth()
+oauth.register(
+    name="google",
+    client_id=settings.GOOGLE_CLIENT_ID,
+    client_secret=settings.GOOGLE_CLIENT_SECRET,
+    server_metadata_url="https://accounts.google.com/.well-known/openid-configuration",
+    client_kwargs={"scope": "openid email profile"},
+)
+
+
+@router.get(
+    "/auth",
+    status_code=status.HTTP_200_OK,
+    responses={**GOOGLE_RESPONSES},  # type: ignore[dict-item]
+)
+async def google_auth(request: Request) -> Any:
```

Comment:
Agregar el `chat_id` como parámetro para que el endpoint ya valide que existe. Si no existe va a devolver un 422 validation error.
No es necesario obtenerlo como `chat_id = request.query_params.get("chat_id")`
Además se agrega automáticamente como parámetro en el swagger/openapi 
Se que antes no estaba asi, pero es la manera correcta de hacerlo :)
```suggestion
async def google_auth(request: Request, chat_id: str) -> Any:
```

Reply:
Ok :)